### PR TITLE
Explain how to run the automation locally

### PR DESCRIPTION
**Pull request by gbenhaim**
Dependent on https://github.com/kubevirt/kubevirt-ansible/pull/255

Signed-off-by: gbenhaim <galbh2@gmail.com>

### CODE REVIEW

File: automation/README.md [hunk 1]
```diff
@@ -71,9 +71,83 @@ are related to specific substage.
 
 The KubeVirt version is static for whole matrix.
 
-# oVirt CI Integration
+## oVirt CI Integration
 
 The [oVirt CI][ovirt-ci-system-doc] project is open to integrating additional
 KubeVirt related projects or integrations, to run per each submitted PR.
 
+## How to run the CI flow locally on my laptop?
```
Comment (shiywang) [inline automation/README.md]: on your laptop or just on laptop ?

Comment (gbenhaim) [reply automation/README.md]: any laptop

File: automation/README.md [hunk 2]
```diff
@@ -71,9 +71,83 @@ are related to specific substage.
...
+run the following command (replace the following variables):
+
+* `jenkins` - the directory that contains the jenkins repository
+  (as explained in [Setting up mock_runner])
+
+* `cluster_version` - The CI flow that you want to run, for example `openshift_3-9.sh`
+
+```bash
+
+  $ <jenkins>/mock_configs/mock_runner.sh \
```
Comment (shiywang) [inline automation/README.md]: prefer `<jenkins>` --> `${jenkins}`

**Comment by shiywang** [comment]
LGTM

File: automation/README.md (hunk 1 above)
Comment (shiywang) [reply automation/README.md]: I would prefer on your laptop cuz it's more consistent with the next sentence, and any seems too wide range, it's not gonna running on any laptop and it has some system requirement
`It's possible to reproduce the CI process on your laptop ...` 

hope my suggestion make sense

**Comment by gbenhaim** [comment]
I've addressed the comments.

**Pull request merged**